docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -50,7 +50,7 @@ Even though .NET Core 3.0 supports **.NET Standard 2.1**, the default `dotnet ne
 </Project>
 ```
 
-If you're using Visual Studio, you need Visual Studio 2019, as Visual Studio 2017 doesn't support **.NET Standard 2.1** or **.NET Core 3.0**. We highly recommend that you use [Visual Studio 2019 Update 1 Preview](https://visualstudio.microsoft.com/vs/preview/).
+If you're using Visual Studio, you need [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019), as Visual Studio 2017 doesn't support **.NET Standard 2.1** or **.NET Core 3.0**.
```

Comment:
Same comment as above.

Reply:
I think here since we're talking about NS 2.1 and .NET Core 3.0 in a general way, I think we don't need to specify preview since they should work since the first version of VS 2019.

@Thraka please confirm.

Reply:
I think this here is OK to have more generic, not specifying a preview.